Dockerfile

```
@@ -21,6 +21,7 @@ RUN <<EOF
     rm -rf /var/lib/apt/lists/*
 EOF
 
+ARG VOICEVOX_ENGINE_REPOSITORY=VOICEVOX/voicevox_engine
 ARG VOICEVOX_ENGINE_VERSION
 ARG VOICEVOX_ENGINE_TARGET
```

あ、ちょっと周りに合わせて初期値なしにさせていただきます！！

```suggestion
ARG VOICEVOX_ENGINE_REPOSITORY
ARG VOICEVOX_ENGINE_VERSION
ARG VOICEVOX_ENGINE_TARGET
```

あ、初期値ですが、なにか値が入っているとデバッグ時等に便利な一方で、デメリットも結構あったりします！

仮に初期値が入っている場合、その意図を探るために使用箇所全部を見て回ったりしないといけないんですよね。
あと仮に初期値がメンテされてなかった場合、変更しても大丈夫なのかを探さないといけない。
あと他に引数名を変えたあと追従できてなかったものがあったり、あるいは引数名を間違えたりしたときにビルドが落ちないから、間違ったものがpushされてしまう。

個人的には、間違ったものが動くくらいならエラーで止まってくれたほうが致命傷を避けられて良いなーと思っているので、引数の初期値はなるべくなくすのが良いかなーと考えています･･･！
まあいろいろな考え方がありますが、ここはとりあえず周囲に合わせてそうしておくという感じで！